aws/scripts/py/s3_sandbox_configuration.py

```
@@ -0,0 +1,65 @@
+import json
+import os
+
+try:
+    BRANCH_NAME  = os.environ['BRANCH_NAME']
+    PROJECT_NAME = os.environ['PROJECT_NAME']
+except KeyError as e:
+    raise ValueError(f"Required environment variable {e} is not set") from e
```

Ruff

Avoid specifying long messages outside the exception class

(TRY003)